operator/v1/types_network.go

```diff
@@ -272,6 +272,14 @@ type OVNKubernetesConfig struct {
 	// +kubebuilder:validation:Minimum=1
 	// +optional
 	GenevePort *uint32 `json:"genevePort,omitempty"`
+	// HybridOverlayExtensionConfig allows us to configure an additional overlay network .
+	// if not specified it is unused
+	HybridOverlayConfig ExtensionConfig `json:"hybridOverlayConfig,omitempty"`
```

Comment:
@JacobTanenbaum if this is optional (as indicated by `omitempty`), please do the following:

1. Add `// +optional` to godoc comment.
2. Make `ExtensionConfig` a pointer.

/cc @squeed

Reply:
yup yup, very good point.

Reply:
Updated PR